webpack.config.js

```diff
@@ -28,9 +25,9 @@ Encore.addPlugin(
 
   .addEntry('bolt', './assets/js/bolt.js')
   .addStyleEntry('theme-default', './assets/scss/themes/default.scss')
-  .addStyleEntry('theme-light', './assets/scss/themes/light.scss')
+  /*.addStyleEntry('theme-light', './assets/scss/themes/light.scss')
```

Comment:
Why have you commented these out?

Reply:
because this is not working with my current changes. it's WIP. anyway thanks for your comments.

Reply:
Question: What do you think about light and dark theme here ? Is it really needed for 'lightweight and simple'? Or isn't creating custom themes already sufficient ?

Reply:
This is for the _backend_, and doesn't really impact the way people _build_ sites with it. Right now, these are "baked in", but this feature will need work for sure. But, it will become better fleshed out sooner or later, so we shouldn't remove them. :-)

Reply:
We need to make way for devs to provide their own admin themes.